src/components/ButtonWithIcon/ButtonWithIcon.scss

```diff
@@ -59,12 +59,12 @@ $block: '.#{$namespace}button-with-icon';
             transition: color 0.2s;
 
             &:hover {
-                color: var(--yc-color-text-link-hover);
+                color: var(--g-color-text-link-hover);
             }
         }
 
         &_accent {
-            color: var(--yc-color-text-yandex-red);
+            color: var(--g-color-text-yandex-red);
```

Comment:
there is the problem with script and this variable, because there is no official replacement for this color, but there is no `--g-color-text-yandex-red`, `--yc-color-text-yandex-red` was just deleted